Formula/qt.rb

```diff
@@ -92,6 +98,9 @@ class Qt < Formula
 
   fails_with gcc: "5"
 
+  # fix build with Xcode 13+
```

Comment:
Should we add an additional comment here reminding us to try to remove this patch in this next release assuming the relevant commits are part of that release.   Presumably if the next release has the commits merged, the patch will fail to apply, but it might be useful to put a remind here anyway.

Reply:
Thanks, @danielnachun, this is good idea. But this will re-trigger a CI for a couple of hours :(

Reply:
If we didn't need to add that other patch we could just make a CI-syntax only follow up.  But since I think that patch is important we can probably add the comment while we're at it.